src/main/java/com/github/lunatrius/schematica/client/util/BlockList.java

```diff
@@ -98,8 +100,34 @@ public String getItemStackDisplayName() {
         }
 
         public String getFormattedAmount() {
+            int placed = this.placed;
+            int placedstack = placed / 64;
+            int placedremainder = placed % 64;
+
+            int total = this.total;
+            int totalstack = total / 64;
+            int totalremainder = total % 64;
+
             final char color = this.placed < this.total ? 'c' : 'a';
-            return String.format("\u00a7%c%d\u00a7r/%d", color, this.placed, this.total);
+            // Add stack count to easier to collect items! "counts(stack:remainder)"
+            return String.format("\u00a7%c%d(%d:%d)\u00a7r/%d(%d:%d)", color, placed, placedstack, placedremainder, total, totalstack, totalremainder);
+        }
+
+        /**
+         * Show you How much items should you collect.
+         * @return require item counts
+         */
+        public String getFormattedCount() {
```

Comment:
The name is pretty much the same as `getFormattedAmount`. `getFormattedAmountRequired` might be a better and more descriptive name.

The math/logic part also seems to be off as well, it doesn't take into account placed blocks at all.